fastddsspy_participants/package.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>fastddsspy_participants</name>
+  <version>0.2.0</version>
+  <description>
+     ...
+  </description>
+  <maintainer email="RaulSanchezMateos@eprosima.com">Raul Sánchez-Mateos</maintainer>
+  <maintainer email="javierparis@eprosima.com">Javier París</maintainer>
+  <maintainer email="juanlopez@eprosima.com">Juan López</maintainer>
+  <license file="LICENSE">Apache 2.0</license>
+
+  <url type="website">https://www.eprosima.com/</url>
+  <url type="bugtracker">https://github.com/eProsima/dev-utils/issues</url>
+  <url type="repository">https://github.com/eProsima/dev-utils</url>
```

Comment:
Change

Reply:
this is a temporary file, this should change in #1